docs/reference/reference-deploy/config/master-config-reference.rst

```diff
@@ -618,6 +618,27 @@ The master supports the following configuration settings:
             such as "30s", "1h", or "1m30s". Valid time units are "s", "m", "h". The default value
             is ``5m``.
 
+      -  ``type: hpc``: Specifies running dynamic agents on HPC clusters. (*Required*)
+
+         -  ``partition``: The target HPC partition where jobs will be launched when using this
+            resource pool. This would usually be used along with ``task_container_defaults`` to
+            specify default job properties, making life easier for users. Consider the following:
```

Comment:
| This would usually be used along with ``task_container_defaults`` to  specify default job properties, making life easier for users.

Add  ``task_container_defaults`` to provide a resource pool with additional default options.   This can be used to create a resource pool with homogeneous resources when the underlying partitions do not.

Reply:
Sounds good to me.